pyomo/solvers/plugins/solvers/CBCplugin.py

```diff
@@ -53,11 +52,13 @@ def configure_cbc():
     if not executable:
         return
     cbc_exec = executable.path()
-    results = run( [cbc_exec,"-stop"], timelimit=1 )
-    _cbc_version = _extract_version(results[1])
-    results = run(
-        [cbc_exec,"dummy","-AMPL","-stop"], timelimit=1 )
-    _cbc_compiled_with_asl = not ('No match for AMPL' in results[1])
+    results = subprocess.run( [cbc_exec,"-stop"], timeout=1,
+                             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    _cbc_version = _extract_version(results.stdout.decode("utf-8"))
+    results = subprocess.run(
+        [cbc_exec,"dummy","-AMPL","-stop"], timeout=1, stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE)
```

Comment:
Is capturing and ignoring stderr a problem here?  Specifically, does '`No match for AMPL'` come out stdout or stderr?